src/form/form-label.style.tsx

```diff
@@ -15,8 +15,7 @@ interface LabelProps {
 export const Label = styled.label<LabelProps>`
```

Comment:
Since disabled styling is no longer needed, can consider removing the style interface entirely.